content/english/Kusto-KC7/activity-1.md

```diff
@@ -0,0 +1,60 @@
+
+
+<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/Resources.png?raw=true" alt= “Resources” width="60%" height="value">
+
+### Setting Up Azure Data Explorer (ADX)
+
+ADX is the primary tool used by some cyber defenders for data exploration and analysis. The great thing about ADX is that it is used by cyber analysts at many of the smallest and largest organizations in the world. 
+
+Let’s get you logged in and started with ADX:
+
+1.	Go to [Azure Data Explorer] https://dataexplorer.azure.com/ and login with your Microsoft account 
+    - If you don't have a Microsoft account already, create one now (they are free to make)
+2.	Click the Query tab on the left side of the screen.
+
+<img src="https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/ADX1.png?raw=true" alt= “ADX1” width="20%" height="value">
+
+Data in ADX is organized in a hierarchical structure which consists of **clusters, databases, and tables**. All of Envolve Labs’s security logs are stored in a single cluster. You’ll need to add this cluster to your ADX interface so you can start looking at the log data
+
+<img src="https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/ADX2.png?raw=true" alt= “ADX2” width="40%" height="value">
+
+3.	Add a new cluster using the cluster URI provided by your instructor
+    -  Click add cluster
+    -  Enter Connection URI: mstictraining.eastus
+
+<img src="https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/ADX3.png?raw=true" alt= “ADX3” width="40%" height="value">
+
+4.	Select your database     
+    - Expand the dropdown arrow next to your cluster. You should then see one database, called **SecurityLogs** inside it.     
+    - Expand the dropdown arrow next to the **SecurityLogs** database.     
+    - Click on the **SecurityLogs** database. Once you’ve done this, you should see the database highlighted- this means you’ve selected the database and are ready to query the tables inside.      
+
+<img src="https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/ADX4_updated.png?raw=true" alt= “ADX4_updated” width="30%" height="value">
+
+The big blank space to the right of your cluster list is the query workspace. That’s where you will use KQL code to write what we call queries, which are used to interact with our log data. 
+
+<img src="https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/ADX5.png?raw=true" alt= “ADX5” width="80%" height="value">
+
+### Scoreboard Setup
+
+1.  After going to the [Scoreboard website] ( https://aka.ms/kc7scoreboard ), create a user account. Be sure to use a username and password you can remember as it will then immediately ask you to login.
```

Comment:
```suggestion
1.  After going to the [Scoreboard website](https://aka.ms/kc7scoreboard), create a user account. Be sure to use a username and password you can remember as it will then immediately ask you to login.
```

Reply:
So doing this edit opened to a blank bing sheet. Apparently the () without spaces between them and the link don't work. But when i took away the () completely they open in a new tab so i think this is the right combo. I've done this for ADX and Scoreboard links.